app/forms/state_file/id_health_insurance_premium_form.rb

```diff
@@ -0,0 +1,21 @@
+module StateFile
+  class IdHealthInsurancePremiumForm < QuestionsForm
+    set_attributes_for :intake, :has_health_insurance_premium, :health_insurance_paid_amount
+
+    validates :has_health_insurance_premium, inclusion: { in: %w[yes no], message: :blank }
+    validates :health_insurance_paid_amount,
+      presence: true,
+      numericality: {
+        allow_blank: true,
+        greater_than_or_equal_to: 0,
```

Comment:
open question whether we should allow `0` value for this amount, but should be ok to review

Reply:
ok to allow 0 here!
>1- it’s possible that the filer may think that they paid Health Insurance Premiums when they initially respond to the question…..but as they more carefully review all of the supporting text in the reveals describing which Health Insurance payments actually qualify for this subtraction they may realize that the Health Insurance Premium payments that they made don’t actually qualify. So allowing them to enter 0 (and filling the PDF field and updating the state XML field with it) is the behavior that we are looking for here.  

https://codeforamerica.atlassian.net/browse/FYST-791?focusedCommentId=12716

Reply:
if they answer no for health insurance premium question, should we still be adding 0 for the xml and pdf?

Reply:
ooh i'll ask this